### ISSUE TITLE
Does not install additional TWBT graphics files

### ISSUE THREAD

**Issue by theasp** [McArcady/lnp-forge#15]
I think the additional files for TWBT support in graphics are not being installed properly.  This is making shadows not work correctly, missing tiles for up/down stairs, etc.  I think LNP does this correctly.

These instructions are from the README.md for Phoebus, and work for me when I do them by hand:
> 2.1.1 - Installing all the TWBT content:  
> To install all the TWBT content, 
> * Copy the contents of the /data/twbt\_art/ folder to the /data/art/.
> * Copy the contents of the /data/twbt\_init/ folder to the /data/init/.
> * Copy the contents of the /raw/twbt\_graphics/ folder to the /raw/graphics/.
> * Copy the contents of the /raw/twbt\_objects/ folder to the /raw/objects/.
> * Copy the "onLoad\_gfx\_Phoebus.init" file into the /raw/ folder.

**Comment by McArcady** [comment]
Thanks for reporting this.
Apart from installing them by hand, did you manage to make PythonLNP install the TWBT-related graphic files correctly ?
From the code, its seems that PythonLNP is able to install files like `shadows.png`  from 'LNP/Tilesets/*', but not from 'LNP/Graphics/*'.
lnp-forge does not install anything in `LNP/Tilesets`, maybe the problem comes from here.

**Comment by theasp** [comment]
It looks like `core/graphics.py` should be handling this here:
https://github.com/Pidgeot/python-lnp/blob/f4c39a2410a8d7171a131619e5e13468506646b2/core/graphics.py#L162

BTW, 0.14a came out 3 days ago, `graphics.py` is unchanged.

**Comment by theasp** [comment]
I added some debugging, it looks like lnp.settings.printmode is `2D` when it should be `TWBT` as per the GUI.

```
INFO: Loading colorscheme ./LNP/Graphics/Phoebus/data/init/colors.txt
INFO: Starting TWBT section
INFO: Removing overrides.txt
INFO: Copying overrides.txt
WARNING: Unable to copy overrides.txt [This looks to be normal, the file is in twbt_init, not init]
INFO: Print mode is 2D
```

**Comment by theasp** [comment]
This seems to be the cause:  https://github.com/Pidgeot/python-lnp/issues/11

Not sure why it's closed, but it does match my results.

**Comment by McArcady** [comment]
to be reproduced with a506009

**Comment by theasp** [comment]
Still an issue as of 0.47.05-r2, which I am guessing has that commit.  Could this be easily fixed by adding an additional copy of Phoebus (and the others) to the menu with the TWBT changes already merged in?  i.e. add "Phoebus (TWBT)".

**Comment by clinodev** [comment]
Just a heads up, there's some movement towards a fork to address this and some other issues with pylnp. I'll keep you up to date. :)

**Comment by McArcady** [comment]
fixed with #53
